### PR TITLE
Bugfix: bad comment

### PR DESCRIPTION
**Pull request by djmetzle**
CC @danielbeardsley 

qa_req 0

### CODE REVIEW

File: frontend/pages/Parts/index.tsx [hunk 1]
```diff
@@ -35,7 +35,7 @@ export const getServerSideProps: GetServerSideProps<AppPageProps> = async (
          getStoreByCode('us'),
          findProductList({ handle: { eq: 'Parts' } }),
       ]);
-
+secondssecondsseconds
```
Comment (hcutrone) [inline frontend/pages/Parts/index.tsx:38]: Was this intentional?

Comment (djmetzle) [reply frontend/pages/Parts/index.tsx:38]: Nope! :sweat_smile:

Comment (djmetzle) [inline frontend/pages/Parts/index.tsx:38]:
```suggestion
```

File: frontend/pages/Parts/index.tsx [hunk 2]
```diff
@@ -35,7 +35,6 @@ export const getServerSideProps: GetServerSideProps<AppPageProps> = async (
          getStoreByCode('us'),
          findProductList({ handle: { eq: 'Parts' } }),
       ]);
-
    if (productList == null) {
```
Comment (djmetzle) [inline frontend/pages/Parts/index.tsx:38]:
```suggestion

   if (productList == null) {
```

File: frontend/pages/Parts/index.tsx [hunk 3]
```diff
@@ -22,7 +22,7 @@ type AppPageProps = WithProvidersProps<PageProps>;
 export const getServerSideProps: GetServerSideProps<AppPageProps> = async (
    context
 ) => {
-   // The data is considered fresh for 10 seconds, and can be served even if stale for up to 10 minutes
+   // The data is considered fresh for 10 minutes, and can be served even if stale for up to 10 minutes
```
Comment (dhmacs) [inline frontend/pages/Parts/index.tsx:25]:
```suggestion
   // The data is considered fresh for 10 minutes, and can be served even if stale for up to 20 minutes
```

**Review by dhmacs — CHANGES_REQUESTED** [review]
Thanks for the PR @djmetzle 👍   
Can you also update the comment on all the other product lists pages?
Also, since we're using Vercel maybe now we can revert [this](https://github.com/iFixit/react-commerce/pull/341)? Vercel does support Stale While Revalidate header, so we can take advantage of that and serve fresh data while still caching it for a while to avoid performance issues

**Comment by djmetzle** [comment]
Sure, I can update the rest of the comments too.

File: frontend/pages/Parts/[...deviceHandleItemType].tsx [hunk 4]
```diff
@@ -27,7 +27,7 @@ type AppPageProps = WithProvidersProps<PageProps>;
 export const getServerSideProps: GetServerSideProps<AppPageProps> = async (
    context
 ) => {
-   // The data is considered fresh for 10 seconds, and can be served even if stale for up to 10 minutes
+   // The data is considered fresh for 10 minues, and can be served even if stale for up to 20 minutes
```
Comment (sterlinghirsh) [inline frontend/pages/Parts/[...deviceHandleItemType].tsx:30]:
```suggestion
   // The data is considered fresh for 10 minutes, and can be served even if stale for up to 20 minutes
```

File: frontend/pages/Shop/[handle].tsx [hunk 5]
```diff
@@ -23,7 +23,7 @@ type AppPageProps = WithProvidersProps<PageProps>;
 export const getServerSideProps: GetServerSideProps<AppPageProps> = async (
    context
 ) => {
-   // The data is considered fresh for 10 seconds, and can be served even if stale for up to 10 minutes
+   // The data is considered fresh for 10 minues, and can be served even if stale for up to 20 minutes
```
Comment (sterlinghirsh) [inline frontend/pages/Shop/[handle].tsx:26]:
```suggestion
   // The data is considered fresh for 10 minutes, and can be served even if stale for up to 20 minutes
```

File: frontend/pages/Tools/index.tsx [hunk 6]
```diff
@@ -22,7 +22,7 @@ type AppPageProps = WithProvidersProps<PageProps>;
 export const getServerSideProps: GetServerSideProps<AppPageProps> = async (
    context
 ) => {
-   // The data is considered fresh for 10 seconds, and can be served even if stale for up to 10 minutes
+   // The data is considered fresh for 10 minues, and can be served even if stale for up to 20 minutes
```
Comment (sterlinghirsh) [inline frontend/pages/Tools/index.tsx:25]:
```suggestion
   // The data is considered fresh for 10 minutes, and can be served even if stale for up to 20 minutes
```

**Review by hcutrone — DISMISSED** [review]
CR :burrito:

**Comment by danielbeardsley** [comment]
I'm a proponent of just deleting the comments. They just say in english what the code already says, nothing more. 

Keeping them up to date has already sucked up a fair amount of time.

Thoughts?

**Comment by djmetzle** [comment]
@danielbeardsley 
I totally agree. The comments currently _lie_, and really do not add much. An [`MDN` link to the cache-control spec](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control) might be more helpful. Worse, those comments are duplicated a half dozen times.

**Comment by danielbeardsley** [comment]
CR :+1:

**Review by danielbeardsley — APPROVED** [review]
CR :+1:

**Pull request merged**